### PR TITLE
Add project-edition-2021.

### PR DESCRIPTION
**Pull request by m-ou-se**
@nikomatsakis @rylev

### CODE REVIEW

File: teams/wg-edition-2021.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name = "wg-edition-2021"
+subteam-of = "lang"
+kind = "working-group"
```
Comment (m-ou-se) [inline teams/wg-edition-2021.toml:3]: Should this be a working-group or project-group or (sub)team?

Comment (rylev) [reply teams/wg-edition-2021.toml:3]: I believe according to [rfc 2856](https://rust-lang.github.io/rfcs/2856-project-groups.html), working groups are for domains and project groups are for specific efforts. Given that, this should be a project group.

Comment (m-ou-se) [reply teams/wg-edition-2021.toml]: Updated!

**Review by nikomatsakis — APPROVED** [review]
Looks good!

**Comment by m-ou-se** [comment]
@pietroalbini Can you give this team access to edit issues on `rust-lang/rust`? Niko and I can already do that, but it'd be useful if Ryan could edit/update tracking issues too.

**Comment by pietroalbini** [comment]
Sure! I need https://github.com/rust-lang/team/pull/544 to be merged before the GitHub team is created though.

**Comment by pietroalbini** [comment]
Granted access!

**Pull request merged**